backend/homework/3/src/main/java/org/example/hospitalmanagement/Doctor.java

```diff
@@ -0,0 +1,16 @@
+package org.example.hospitalmanagement;
```

Comment:
Pls add a print method in your class for entities or override the `toString()` menthod.